zendesk.model.lkml

```diff
@@ -215,6 +215,12 @@ explore: tickets {
     relationship: many_to_one
   }
 
+  join: ticket_metrics {
+    type: left_outer
+    sql_on: ${tickets.id} = ${ticket_metrics.ticket_id} ;;
+    relationship: one_to_one
+  }
```

Comment:
👍   

Please test the new JOIN as well as the new dimensions/measures and make sure they work as expected. Thanks!